modules/virt-configuring-openshift-root-CA-for-providers.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * virt/live_migration/virt-about-mtv-providers.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-configuring-openshift-root-CA-for-providers_{context}"]
+= Configuring the OpenShift root certificate authority for providers
+
+You must configure an {product-title} provider for each cluster that you are including in the migration, and each provider requires a certificate authority (CA) for the cluster. It is important to configure the root CA for the entire cluster to avoid a CA's early expiration, which causes the provider to fail.
+
+.Procedure
+
+. Run the following command against the cluster for which you are creating the provider:
++
+[source,terminal]
+----
+$ oc get cm kube-root-ca.crt -o=jsonpath={.data.ca\\.crt}
+----
+
+. Paste the output into the provider creation form, as shown in the following example:
++
+[source,terminal]
+----
+-----BEGIN CERTIFICATE-----
+<CA_certificate_content>
+-----END CERTIFICATE-----
+----
```

Comment:
It feels like there should be a third step here, but that isn't really in the scope of this peer review(-ish).